changelog/@unreleased/pr-6467.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: Update slf4j javadocs link to https://www.slf4j.org/apidocs/
```

Comment:
we just _removed_ it right?

Reply:
Updated. I had first tried switch from http to https but I think the issue is CI trying to egress, so ended up removing it for now.